frappe/utils/background_jobs.py

```diff
@@ -209,22 +212,37 @@ def execute_job(site, method, event, job_name, kwargs, user=None, is_async=True,
 			frappe.destroy()
 
 
-def start_worker(queue=None, quiet=False, rq_username=None, rq_password=None):
+def start_worker(
+	queue: str | None = None,
+	quiet: bool = False,
+	rq_username: str | None = None,
+	rq_password: str | None = None,
+	burst: bool = False,
+	strategy: Literal["round_robbin", "random"] | None = None,
+) -> NoReturn | None:
 	"""Wrapper to start rq worker. Connects to redis and monitors these queues."""
+	DEQUEUE_STRATEGIES = {"round_robbin": RoundRobinWorker, "random": RandomWorker}
```

Comment:
typo?
```suggestion
	DEQUEUE_STRATEGIES = {"round_robin": RoundRobinWorker, "random": RandomWorker}
```

Reply:
wat, I thought I had pushed this fix but ...
![image](https://user-images.githubusercontent.com/9079960/203926768-648c6bd3-6775-47a0-b598-96e96e829845.png)